net.zgcoder.skycheckers.json

```diff
@@ -0,0 +1,136 @@
+{
+	"app-id": "net.zgcoder.skycheckers",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "skycheckers",
+	"finish-args": [
+		"--socket=x11", "--share=ipc",
+		"--device=dri", "--socket=pulseaudio", "--share=network", "--device=all"
+	],
+	"modules": [
+		{
+			"name": "SDL2",
+			"buildsystem": "autotools",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libsdl.org/release/SDL2-2.0.12.tar.gz",
+					"sha256": "349268f695c02efbc9b9148a70b85e58cefbbf704abd3e91be654db7f1e2c863"
+				}
+			],
+			"cleanup": ["/bin/sdl2-config", "/include", "/lib/libSDL2.la", "/lib/libSDL2main.a", "/lib/libSDL2main.la", "/lib/libSDL2_test.a", "/lib/libSDL2_test.la", "/lib/cmake", "/share/aclocal"]
+		},
+		{
+			"name": "SDL2_mixer",
+			"buildsystem": "autotools",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.0.4.tar.gz",
+					"sha256": "b4cf5a382c061cd75081cf246c2aa2f9df8db04bdda8dcdc6b6cca55bede2419"
+				}
+			],
+			"cleanup": ["/lib/libSDL2_mixer.la", "/include"]
+		},
+		{
+			"name": "SDL2_ttf",
+			"buildsystem": "autotools",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.libsdl.org/projects/SDL_ttf/release/SDL2_ttf-2.0.15.tar.gz",
+					"sha256": "a9eceb1ad88c1f1545cd7bd28e7cbc0b2c14191d40238f531a15b01b1b22cd33"
+				}
+			],
+			"cleanup": ["/lib/libSDL2_ttf.la", "/include"]
+		},
+		{
+			"name": "glu",
+			"config-opts": ["--disable-static"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://mesa.freedesktop.org/archive/glu/glu-9.0.1.tar.xz",
+					"sha256": "fb5a4c2dd6ba6d1c21ab7c05129b0769544e1d68e1e3b0ffecb18e73c93055bc"
+				}
+			],
+			"cleanup": ["/include", "/lib/*.a", "/lib/*.la", "/lib/pkgconfig"]
+		},
+		{
+			"name": "glew",
```

Comment:
Any reason for not using the one available in shared-modules https://github.com/flathub/shared-modules?

Reply:
Added.